File: EjerciciosClase/22-2-2019/cpu.py
import math

MAX = 100000000

def main():
    i, seno, coseno, tangente, logaritmo, raizCuadrada = 1, 1, 1, 1, 1, 1
    while(i < MAX):
        seno += math.sin(i)
        coseno += math.cos(i)
        tangente += math.tan(i)
        logaritmo += math.log(i)
        raizCuadrada += math.sqrt(i)
        i += 1

if __name__ == "__main__":
    main()

File: EjerciciosClase/22-2-2019/cpu.java
import java.lang.*;
public class cpu {

   /* This is my first java program.
    * This will print 'Hello World' as the output
    */

   public static void main(String []args) {
        double MAX = 100000000;
        double i = 0;
        double seno = 0;
        double coseno = 0;
        double tangente = 0;
        double logaritmo = 0;
        double raizCuadrada = 0;
        while(i < MAX){
            seno += Math.sin(i);
            coseno += Math.cos(i);
            tangente += Math.tan(i);
            logaritmo += Math.log(i);
            raizCuadrada += Math.sqrt(i);
            i++;
        }
   }
}